.travis.yml

```diff
@@ -21,7 +21,9 @@ matrix:
         - python: 3.5
           env: TOXENV=py35
         - python: pypy
-          env: TOXENV=pypy
+          env: TOXENV=pypy PYPY_VERSION=4.0.1
```

Comment:
I believe we actually claim to support pypy 2.6, so if we want oldest/newest this should probably become 2.6?